integration-tests/types/config/node/core.go

```diff
@@ -116,6 +119,33 @@ func SetDefaultSimulatedGeth(cfg *chainlink.Config, ws, http string) {
 	}
 }
 
+func SetChainConfig(cfg *chainlink.Config, wsUrls, httpUrls []string, chain blockchain.EVMNetwork) {
```

Comment:
Should we remove `SetDefaultSimulatedGeth` then? One method to configure both simulated and non-simulated.